pkg/strategy/grid2/strategy.go

```diff
@@ -414,19 +441,7 @@ func (s *Strategy) processFilledOrder(o types.Order) {
 		profit = s.calculateProfit(o, newPrice, newQuantity)
 
 	case types.SideTypeBuy:
-		// baseSellQuantityReduction calculation should be only for BUY order
-		// because when 1.0 BTC buy order is filled without FEE token, then we will actually get 1.0 * (1 - feeRate) BTC
-		// if we don't reduce the sell quantity, than we might fail to place the sell order
-		baseSellQuantityReduction = s.aggregateOrderBaseFee(o)
-		s.logger.Infof("GRID BUY ORDER BASE FEE: %s %s", baseSellQuantityReduction.String(), s.Market.BaseCurrency)
-
-		baseSellQuantityReduction = roundUpMarketQuantity(s.Market, baseSellQuantityReduction)
-		s.logger.Infof("GRID BUY ORDER BASE FEE (Rounding with precision %d): %s %s",
-			s.Market.VolumePrecision,
-			baseSellQuantityReduction.String(),
-			s.Market.BaseCurrency)
-
-		newQuantity = newQuantity.Sub(baseSellQuantityReduction)
+		newQuantity = newQuantity.Sub(feeQuantityReduction)
```

Comment:
Need to check fee is in quote or base currency here?

Reply:
You're right

Reply:
Fixed